do.sh

```diff
@@ -348,7 +398,10 @@ function run_test() {
     for f in *.tgz; do
        tar xvfz $f
     done
-    popd
+    # Prior to containerization of ovn-tester, HTML files written by ovn-tester
+    # were written directly to ${out_dir}. To make things easier for tools, we
+    # copy the HTML files back to this original location.
+    cp ${tester_host}/ovn-tester/*.html ${out_dir}
```

Comment:
If the test failed and there are no html files, this command will fail the whole ./do.sh because we're running with errexit.  This way we will loose the data mining stage.

Reply:
I resolved this by moving the `cp` into `mine_data()` so that we don't exit if the HTML files are not present.